src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -49,6 +49,7 @@ struct RenderRepeater
     uint8_t endOpacity;
     bool interpOpacity;
     bool inorder;
+    bool external;
```

Comment:
We strongly recommend adding a comment for this kind of ambiguous data.

Reply:
I changed the approach, since I've found nested examples. I need to know whether the level of repeater was greater or it was equal to the level of repeated shape